Makefile

```diff
@@ -14,7 +14,7 @@ LIBRARY_PATH=
 REPORT_ARG=
 REPOSITORIES ?= $(shell ls $(_SYSUPG_REPOS) | xargs echo | tr " " ",")
 SYSUPG_TEST_PATHS=$(shell echo $(REPOSITORIES) | sed -r "s|(,\\|^)| $(_SYSUPG_REPOS)/|g")
-TEST_PATHS:=commands repos/common $(SYSUPG_TEST_PATHS)
+TEST_PATHS:=commands repos/common tests/commands $(SYSUPG_TEST_PATHS)
```

Comment:
@fernflower  move `tests/commands` under `commands/tests` and drop this change. That's the path where we expect tests for commands right now (the spec reflects that as well).

Reply:
Hmm ok, wasn't aware we had agreed on command tests path.